browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2953,6 +2953,8 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_PERMISSION_WAS_CANCELLED" desc="Vpn user consent text">
       Permission was cancelled
     </message>
+    <message name="IDS_BRAVE_WALLET_QUEUE_REJECT_ALL" desc="Confirm Transaction Queue Reject All Transactions">Reject <ph name="TRANSACTION_LENGTH"><ex>4</ex>%1$s</ph> transactions</message>
+
```

Comment:
nit : I don't think we require new line here.

Reply:
could you please format it also as all strings formatted?

Reply:
done